src/integrationTest/sam.test.ts

```diff
@@ -314,7 +314,14 @@ async function startDebugger(
     // Executes the 'F5' action
     await vscode.debug.startDebugging(undefined, testConfig).then(
         async () => {
-            logSession('START', vscode.debug.activeDebugSession!.name)
+            const session: vscode.DebugSession | undefined = await waitUntil(
```

Comment:
This is causing a race condition somewhere I guess ??

Reply:
I guess it isn't failing correctly. Tests are timing out:

```
Exceeded timeout of 120.0 seconds: "SAM Integration Tests SAM Application Runtime: nodejs14.x (Image) Starting with a newly created nodejs14.x (Image) SAM Application... target=api: invokes and attaches on debug request (F5)": AssertionError [ERR_ASSERTION]: Exceeded timeout of 120.0 seconds: "SAM Integration Tests SAM Application Runtime: nodejs14.x (Image) Starting with a newly created nodejs14.x (Image) SAM Application... target=api: invokes and attaches on debug request (F5)"
    at Timeout.<anonymous> (/codebuild/output/src172959586/src/github.com/aws/aws-toolkit-vscode/src/integrationTest/globalSetup.test.ts:44:16)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)

Reply:
Some of the run/debug flows terminate almost immediately if there's no break point set. We're probably missing those and thus locking up.

Reply:
Could try adding a `SourceBreakpoint` via vscode.debug.addBreakpoints to the first line in each file?
Something like:
```ts
const bp = new vscode.SourceBreakpoint(new vscode.Location(sourceFileUri, new vscode.Position(1, 1), true))
vscode.debug.addBreakpoints([bp])
```
Prior to starting the debug session. Idk how well this would work across multiple languages.